tests/test-rankprofiles.C

```diff
@@ -167,13 +170,13 @@ bool run_with_field(Givaro::Integer q, uint64_t b, size_t m, size_t n, size_t r,
             // for (size_t i=0; i<r; i++)
             // 	F->assign(RPM[RRP[i]*n+CRP[i]],F->one);
             fassign (*F, n, n, A, lda, B, lda);
-            size_t cs = FFPACK::ColumnRankProfile (*F, n, n, A, lda, CRPLUD, FFPACK::FfpackSlabRecursive);
+            size_t cs = FFPACK::ColumnRankProfile (*F, n, n, A, lda, CRPLUD, FFPACK::FfpackSlabRecursive, seqH);
```

Comment:
Your choice TileRecursive -> parallel, SlabRecursive -> sequential is arbitrary. Please restore all these calls to calls without helper, and add additional calls with parallel helpers after these.

Reply:
Note also that SlabRecursive calls LUdivine which has no parallel version, so you never test the parallel code

Reply:
I added one sequential helper and one without any helper as before to test if the modification for all possible use of RankProfile but I will revert this if you like